### ISSUE TITLE
torch.cross outputs incorrect results

### ISSUE THREAD

**Issue by HotzingTone** [pytorch/pytorch#76588]
### 🐛 Describe the bug

torch.cross output incorrect / imprecise results

e.g.
torch.cross(torch.tensor([4.294968e+09, 4.294968e+09, 4.294968e+09]), torch.tensor([3., 3., 3.]))
-> tensor([-512., -512., -512.])

torch.cross(torch.tensor([4.294968e+05, 4.294968e+05, 4.294968e+05]), torch.tensor([3., 3., 3.]))
-> tensor([-0.0625, -0.0625, -0.0625])

The correct results for both should be tensor([0., 0., 0.])

### Versions

1.9.0

**Comment by ngimel** [comment]
This is expected behavior due to finite floating point accuracy.
